src/localization/ru/learn.json

```diff
@@ -6,7 +6,7 @@
   "examples.finish.description": " ",
 
   "steps.starter.title": "Вступление",
-  "steps.starter.description": "RegEx - это сокращение от `Regular Expression`. Это помогает сопоставлять, находить или управлять текстом. Начните с ввода `OK` в поле RegEx, чтобы перейти к первому шагу и получить доступ к более подробному описанию..",
+  "steps.starter.description": "RegEx - это сокращение от `Regular Expression` `(Регулярное выражение)`. Это помогает сопоставлять, находить или управлять текстом. Начните с ввода `OK (латиницей)` в поле RegEx, чтобы перейти к первому шагу и получить доступ к более подробному описанию..",
```

Comment:
Maybe this is better. Because I'm just trying to highlight what needs to be written.
```
`OK` (латиницей)
```

Reply:
Sure, makes sense